CampaignAutomation/Tests/CampaignAutomationImplementingTests.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  CampaignAutomationImplementingTests.swift
+//  RealifeTechTests
+//
+//  Created by Ross Patman Work on 08/12/2021.
+//  Copyright © 2021 Realife Tech. All rights reserved.
+//
+
+import XCTest
+@testable import RealifeTech
+
+class CampaignAutomationImplementingTests: XCTestCase {
+
+    private var mockAnalytics: MockAnalytics!
+    private var sut: CampaignAutomationImplementing!
+
+    override func setUpWithError() throws {
+        mockAnalytics = MockAnalytics()
+        sut = CampaignAutomationImplementing(defaultFetcher: RLTViewFetcher(),
+                                             analyticsLogger: mockAnalytics)
+    }
+
+    override func tearDownWithError() throws {
+        mockAnalytics = nil
+        sut = nil
```

Comment:
Could we call `try super. tearDownWithError() ` please? Thank you.
The same reason as above.

Reply:
Done.